### PR TITLE
Update readme.md

### PR DESCRIPTION
**Pull request by TheEhsanSarshar**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -17,6 +17,10 @@ You can try a live demo [here](https://codesandbox.io/s/dazzling-moon-itop4).
 ```bash
 npm install zustand
 ```
+#### or
+```bash
+yarn add zustand
+```
```
Comment (dai-shi) [inline readme.md:23]: we prefer to slim readme as much as possible. do you have an idea to make it compact? how about this?
```bash
npm install zustand # or yarn add zustand
```

Comment (TheEhsanSarshar) [reply readme.md:23]: Ok, I will do that.
but still, the readme is too heavy.
we should have a better doc

File: readme.md [hunk 2]
```diff
@@ -15,9 +15,8 @@ Don't disregard it because it's cute. It has quite the claws, lots of time was s
 You can try a live demo [here](https://codesandbox.io/s/dazzling-moon-itop4).
 
 ```bash
-npm install zustand
+npm install zustand # or yarn add zustand
 ```
-
 ## First create a store
```
Comment (dai-shi) [inline readme.md:20]:
```suggestion

## First create a store
```

**Review by dai-shi — APPROVED** [review]
The change is okay. Let's wait for fixing tests before merge.

**Pull request merged**